base/ltfssaxes.dtx

```diff
@@ -192,503 +250,3177 @@
+\DeclareFontSeriesChangeRule {ux}{ub}   {ubux} {bux}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbuc}{ul} {uluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{el} {eluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{l}  {luc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{sl} {sluc} {}
+\DeclareFontSeriesChangeRule {sbuc}{sb} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbuc}{b}  {buc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{eb} {ebuc} {buc}
+\DeclareFontSeriesChangeRule {sbuc}{ub} {ubuc} {buc}
+\DeclareFontSeriesChangeRule {sbuc}{uc} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbuc}{ec} {sbec} {}
+\DeclareFontSeriesChangeRule {sbuc}{c}  {sbc}  {}
+\DeclareFontSeriesChangeRule {sbuc}{sc} {sbsc} {}
+\DeclareFontSeriesChangeRule {sbuc}{sx} {sbsx} {}
+\DeclareFontSeriesChangeRule {sbuc}{x}  {sbx}  {}
+\DeclareFontSeriesChangeRule {sbuc}{ex} {sbex} {}
+\DeclareFontSeriesChangeRule {sbuc}{ux} {sbux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbec}{ul} {ulec} {}
+\DeclareFontSeriesChangeRule {sbec}{el} {elec} {}
+\DeclareFontSeriesChangeRule {sbec}{l}  {lec}  {}
+\DeclareFontSeriesChangeRule {sbec}{sl} {slec} {}
+\DeclareFontSeriesChangeRule {sbec}{sb} {sbec} {bec}
+\DeclareFontSeriesChangeRule {sbec}{b}  {bec}  {}
+\DeclareFontSeriesChangeRule {sbec}{eb} {ebec} {bec}
+\DeclareFontSeriesChangeRule {sbec}{ub} {ubec} {bec}
+\DeclareFontSeriesChangeRule {sbec}{uc} {sbuc} {}
+\DeclareFontSeriesChangeRule {sbec}{ec} {sbec} {}
+\DeclareFontSeriesChangeRule {sbec}{c}  {sbc}  {}
+\DeclareFontSeriesChangeRule {sbec}{sc} {sbsc} {}
+\DeclareFontSeriesChangeRule {sbec}{sx} {sbsx} {}
+\DeclareFontSeriesChangeRule {sbec}{x}  {sbx}  {}
+\DeclareFontSeriesChangeRule {sbec}{ex} {sbex} {}
+\DeclareFontSeriesChangeRule {sbec}{ux} {sbux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbc}{ul}  {ulc}  {}
+\DeclareFontSeriesChangeRule {sbc}{el}  {elc}  {}
+\DeclareFontSeriesChangeRule {sbc}{l}   {lc}   {}
+\DeclareFontSeriesChangeRule {sbc}{sl}  {slc}  {}
+\DeclareFontSeriesChangeRule {sbc}{sb}  {sbc}  {}
+\DeclareFontSeriesChangeRule {sbc}{b}   {bc}   {}
+\DeclareFontSeriesChangeRule {sbc}{eb}  {ebc}  {bc}
+\DeclareFontSeriesChangeRule {sbc}{ub}  {ubc}  {bc}
+\DeclareFontSeriesChangeRule {sbc}{uc}  {sbuc} {sbc}
+\DeclareFontSeriesChangeRule {sbc}{ec}  {sbec} {}
+\DeclareFontSeriesChangeRule {sbc}{c}   {sbc}  {}
+\DeclareFontSeriesChangeRule {sbc}{sc}  {sbsc} {}
+\DeclareFontSeriesChangeRule {sbc}{sx}  {sbsx} {}
+\DeclareFontSeriesChangeRule {sbc}{x}   {sbx}  {}
+\DeclareFontSeriesChangeRule {sbc}{ex}  {sbex} {}
+\DeclareFontSeriesChangeRule {sbc}{ux}  {sbux} {}
+%    \end{macrocode}
+%
+%    \begin{macrocode}
+\DeclareFontSeriesChangeRule {sbsc}{ul} {ulsc} {}
+\DeclareFontSeriesChangeRule {sbsc}{el} {elsc} {}
+\DeclareFontSeriesChangeRule {sbsc}{l}  {lsc}  {}
+\DeclareFontSeriesChangeRule {sbsc}{sl} {slsc} {}
+\DeclareFontSeriesChangeRule {sbsc}{sb} {sbsc} {bsc}
```

Comment:
```suggestion
\DeclareFontSeriesChangeRule {sbsc}{sb} {sbsc} {}
```

Another rule with a useless fallback because the third argument is the same as the first.